FearlessUtils/Classes/Crypto/JunctionFactory.swift

```diff
@@ -22,6 +22,7 @@ public enum JunctionFactoryError: Error {
     case multiplePassphrase
     case emptyJunction
     case invalidStart
+    case invalidBIP32Junction
```

Comment:
Let's create a separate error enum for BIP32

Reply:
Done